src/components/albums/NewAlbum.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <form @submit.prevent="handleSubmit">
+    <input v-model="album.title" required>
+    <button>Add</button>
+  </form>
+</template>
+
+<script>
+import albumsApi from '../../services/albumsApi';
+
+export default {
+    data() {
+        return {
+            album: {
+                title: ''
```

Comment:
```suggestion
                title: '',
                images: []
```
For Vue to work its "magic" properly, you need to instantiate the empty array on the album object. This will allow Vue to "track" data changes and update the render. Right now, if you add an image to a newly created album, you have to navigate away from the list and then back to it to see your new image. By instantiating the empty array, when you add the image to it, it will render immediately.